### ISSUE TITLE
Feature request: Show property type on hover

### ISSUE THREAD

**Issue by srg-kostyrko** [JSMonk/hegel#110]
Currently, types are shown on hover only for variables/functions.
It would be great to be able to see type when you hover object property.

This is a sample hover hint in typescript - `(property) A.b: string`.

**Comment by JSMonk** [comment]
Yes. It's a really important feature. We will implement it soon.
Thank you for your contribution ^_^
